hardware/scripts/hardware.js

```diff
@@ -7,44 +8,40 @@ const Hardware = (function Hardware() {
   const Constants = {
     HeaderSize: 2,
     FooterSize: 1,
-    PixelSize: 3,
-    Width: 64,
-    Height: 32,
+    PixelSize: 3, // 3 colors
+    Width: 96,
+    Height: 16,
+    BoardCount: 3, // this needs to be a proper divider of the led count
   };
 
-  const leds = document.getElementsByClassName('led');
-
-  const kLedsCount = (Constants.Width / 2) * Constants.Height;
-  if (leds.length != kLedsCount) {
-    let msg = 'The total number of leds should be ' + kLedsCount + '.\n' +
-              'Current count is ' + leds.lenght + '.';
-    throw new Error(msg);
-  } 
+  let leds;
 
   function updateLed(id, r, g, b) {
-    if (r == 0 && g == 0 && b == 0) {
+    if (r === 0 && g === 0 && b === 0) { // TODO use blend-mode ?
```

Comment:
I think we can try to use mix-blend-mode or background-blend-mode somewhere to better simulate the transparency. but later :)